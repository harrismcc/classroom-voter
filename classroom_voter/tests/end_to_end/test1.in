localhost
1500
student@gmail.com
temp_pswd
new_pswd




vp

