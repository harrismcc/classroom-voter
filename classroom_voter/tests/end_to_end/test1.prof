localhost
1500
Login
prof@gmail.com
temp_pswd
lowercaseUPERCASE1!
lowercaseUPERCASE1!
0
np
What is the answer?
This is the answer.
10:15 01-15-2040



cr
1
quit
