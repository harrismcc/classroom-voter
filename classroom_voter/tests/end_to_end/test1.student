localhost
1500
Login
student@gmail.com
temp_pswd
lowercaseUPERCASE1!
lowercaseUPERCASE1!
0




vp
This is my answer as a student.





quit
