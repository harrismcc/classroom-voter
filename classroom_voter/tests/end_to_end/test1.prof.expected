################################################################################
			Log in to classroom voter
################################################################################
Enter the IP address of the server (eg 192.168.61.1): localhost
Enter the port of the server (eg 1500): 1500
Successful Connection
Login or Forgot Password: Login
Enter username: prof@gmail.com
Enter Password: temp_pswd
please enter your new password: 
(note: Password must have atleast 8 characters including an uppercase and lowercase letter, a symbol, and a digit.
 Password: lowercaseUPERCASE1!
Please enter the password again to confirm: lowercaseUPERCASE1!
Enrolled Courses:
0 : cs181
Select a course id: 0
To create a new poll, enter  'np'. To collect responses, enter 'cr'. To quit, enter 'quit': np
[Required] Enter your poll question: What is the answer?
[Optional] Enter your poll answer: This is the answer.
[Required] Enter the due date (24-hour clock: %H:%M %m-%d-%Y): 10:15 01-15-2040
To create a new poll, enter  'np'. To collect responses, enter 'cr'. To quit, enter 'quit': cr
Collect reponses for pollId: 1
Results:  {'endpoint': 'Aggregation_result', 'Arguments': {'results': [['{"responseBody": "This is my answer as a student.", "anonLevel": 0}']]}}
To create a new poll, enter  'np'. To collect responses, enter 'cr'. To quit, enter 'quit': quit
################################################################################
			Closing session
################################################################################
